src/sentry/static/sentry/app/components/sidebar/sidebarMenuItemLink.tsx

```diff
@@ -3,23 +3,17 @@ import React from 'react';
 import Link from 'app/components/links/link';
 import ExternalLink from 'app/components/links/externalLink';
 
-export type SidebarMenuItemLinkProps = {
+type Props = {
+  // SidebarMenuItemLink content (accepted via string or components / DOM nodes)
+  children: React.ReactNode;
   /**
    * Use this prop if button is a react-router link
    */
   to?: string;
   /**
-   * Use this prop if button should use a normal (non-react-router) link
+   * Use this prop if button is an external link
    */
   href?: string;
-  /**
-   * Is an external link? (Will open in new tab; Only applicable if `href` is used)
-   */
-  external?: boolean;
-  /**
-   * specifies whether to open the linked document in a new tab
-   */
-  openInNewTab?: boolean;
```

Comment:
This can get passed through to `<ExternalLink>` now, correct?

Reply:
yes